Conformity/custom-rule/custom-rule.py

```diff
@@ -0,0 +1,269 @@
+#!/usr/local/bin/python3
+
+from genericpath import isdir
+import os
+import argparse
+import sys
+from urllib import request, response
+import requests
+import json
+import yaml
+
+configFolder = os.path.expanduser(path='~/.conformity-custom-rule')
+configFilename = 'config.yaml'
+
+workspaceFolder = os.path.expanduser(path='./workspace')
+
+apiEndpoint = "https://conformity.{}.cloudone.trendmicro.com/api/custom-rules"
+
+def readConfiguration():
+	configurationFile = f'{configFolder}/{configFilename}'
+	
```

Comment:
A recommendation for the functions is to add `try` and `except` so errors can be found and troubleshooted easier

Reply:
Exceptions management was added throughout the tool.